packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -210,6 +224,18 @@ export class OdspDelayLoadedDeltaStream {
 		});
 	}
 
+	private readonly signalHandler = (signalsArg: ISignalMessage | ISignalMessage[]) => {
+		const signals = Array.isArray(signalsArg) ? signalsArg : [signalsArg];
+		signals.forEach((signal: ISignalMessage) => {
+			const envelope = JSON.parse(signal.content as string) as ISignalEnvelope;
+			if (envelope.contents.type === "PolicyLabelsUpdate") {
```

Comment:
Can we also check clientId? I think it should be null/undefined (whatever signifies that it's system message), and not accept messages with same type from other clients.
Also, can we have this constant ("PolicyLabelsUpdate") somewhere in "contracts"? I think I've created contracts.ts or something like that.

Reply:
yes, done